tests/js/fixtures/multiterm/searchindex.js

```diff
@@ -0,0 +1 @@
+Search.setIndex({"alltitles": {"Main Page": [[0, "main-page"]]}, "docnames": ["index"], "envversion": {"sphinx": 61, "sphinx.domains.c": 3, "sphinx.domains.changeset": 1, "sphinx.domains.citation": 1, "sphinx.domains.cpp": 9, "sphinx.domains.index": 1, "sphinx.domains.javascript": 3, "sphinx.domains.math": 2, "sphinx.domains.python": 4, "sphinx.domains.rst": 2, "sphinx.domains.std": 2}, "filenames": ["index.rst"], "indexentries": {}, "objects": {}, "objnames": {}, "objtypes": {}, "terms": {"welcom": 0}, "titles": ["Main Page"], "titleterms": {"main": 0, "page": 0}})
```

Comment:
Since it's a fixture, could you make it larger? like more readable? (prettify it). It doesn't hurt for debugging / extending the fixture.

Reply:
Yep, I was thinking about doing this too.  Especially if/when the format of the index changes, that could be very useful.

To do that, we'd want to edit the Python [`_JavaScriptIndex.dumps` code](https://github.com/sphinx-doc/sphinx/blob/6aaeee206bda210f1b2cd845a924f86b18c0af8e/sphinx/search/__init__.py#L161-L165) to set `indent=4` or something similar.  But we wouldn't want that by default for most projects, because it'll increase the file size of the index.

Reply:
I think I'd file a feature request for this: a config setting `html_search_minify_index` (or similar), default `True`.  We would set it to `False` in the configuration files in the JS test roots.

Reply:
Wait, the fixtures are always auto-generated? if so, you can just run a post-processing step where you prettify that specific fixture, right?

Reply:
That could be an easier option, yes!

Reply:
The generation occurs within a Python context (running the Sphinx build) so I am considering https://github.com/beautifier/js-beautify which can be installed using `pip`.

Reply:
(it would be a post-processing step, but even so, we'll want something that can easily run on any developer/CI machine, preferably without installing system-level packages)

Reply:
Ok, for now let's leave it like this

Reply:
Ah, I missed that comment before developing and pushing the change.

Reply:
> Wait, the fixtures are always auto-generated? if so, you can just run a post-processing step where you prettify that specific fixture, right?

I'm going to remove the post-processing again I think, and there's a specific reason I have in mind: by doing that, I can add a Python test that builds each of these roots and confirms that the `searchindex.js` in the JS test directory is correct.

They should change extremely rarely, and if people want to view prettified diffs of those, they can do relatively easily locally.